src/Transition/InputRulesFiles/Rules23-1-0-to-23-2-0.md

```diff
@@ -4,8 +4,339 @@ Input Changes version 23.1.0 to 23.2.0
 This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc.
 This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
 
-# Object Change: OBJECT NAME HERE
+# Key Note
```

Comment:
It would be good for someone to read through this, but not mandatory before merge.  The text could be improved post-IO-freeze if absolutely necessary.